packages/ng/multi-select/input/select-all/multi-select-all-displayer.component.ts

```diff
@@ -30,7 +30,7 @@ import { MULTI_SELECT_WITH_SELECT_ALL_CONTEXT } from './select-all.models';
 							}
 						</div>
 					} @else {
-						<lu-numeric-badge [value]="displayerCount()" />
+						<lu-numeric-badge class="multipleSelect-displayer-numericBadge" [value]="displayerCount()" />
```

Comment:
```suggestion
						<lu-numeric-badge class="multipleSelect-displayer-numericBadge numericBadge" [value]="displayerCount()" />
```

Reply:
With the double class, we can more easily target the element by giving it weight (and without using descendants).

Reply:
It's not about specificity, we already defined this class on Multi Select : https://github.com/LuccaSA/lucca-front/blob/46a8888f0ae9388061a9532ea74659eb1c3c2c12/packages/scss/src/components/multiSelect/mods.scss#L34